metadata.rb

```diff
@@ -14,7 +14,6 @@
 supports 'ubuntu'
 supports 'linuxmint'
 supports 'debian'
-supports 'freebsd'
```

Comment:
nope I was imagining that for some reason. I'll remove it, and note it in the readme, as there's no way we can test it reliably.